cfme/infrastructure/virtual_machines.py

```diff
@@ -895,6 +895,7 @@ def reconfigure(self, new_configuration=None, changes=None, cancel=False):
                      See VMConfiguration.get_changes_to_fill to see expected format of the data
             cancel: `False` if we want to submit the changes, `True` otherwise
         """
+        from cfme.infrastructure.provider.rhevm import RHEVMProvider
```

Comment:
Why not put this with the imports at the top of the file?

Reply:
infinite imports loop ?